docs/CHANGELOG.md

```diff
@@ -5,11 +5,15 @@ Version 3.34.0-eisop2 (May ?, 2023)
 
 **Implementation details:**
 
+Deprecated `ObjectCreationNode#getConstructor` in favor of `ObjectCreationNode#getIdentifier()`.
```

Comment:
change to `getTypeToInstantiate`?

Reply:
Can you also update the information in issue#376?

Reply:
Thanks!